src/components/connection-prompt/ConnectDialogContainer.svelte

```diff
@@ -45,7 +45,9 @@
           btPatternInput.set(MBSpecs.Utility.nameToPattern(friendlyName));
         }
 
-        Microbits.flashHexToLinked(progress => {
+        // radio-local is a temporary debug hack
+        const hexForStage = currentStage === 'usb' ? 'bluetooth' : 'radio-local';
```

Comment:
In future I envisage usb usb1 usb2 being changed to align with bluetooth, radio-sender, radio-bridge values and then radio-local being an alternative that's forced on via a flag.